packages/terra-table/src/proptypes/columnShape.js

```diff
@@ -42,7 +47,19 @@ const columnShape = PropTypes.shape({
    * One of `ascending`, `descending`.
    */
   sortIndicator: PropTypes.oneOf(Object.values(SortIndicators)),
+
+  /**
+   * @private
+   * The color to be used for highlighting a column.
+   */
+  columnHighlightColor: PropTypes.oneOf(Object.values(ColumnHighlightColor)),
+
+  /**
+   * @private
+   * The information to be conveyed to screen readers about the highlighted column.
+   */
+  columnHighlightDescription: PropTypes.string,
 });
```

Comment:
These props should not be private.

Reply:
Changed in https://github.com/cerner/terra-framework/pull/1985/commits/fce4b663063a694bddaeaf64d6bd4ce240c1020f